packages/cli/package.json

```diff
@@ -1,9 +1,36 @@
 {
   "name": "@code-pushup/cli",
   "version": "0.5.0",
+  "license": "MIT",
   "bin": {
     "code-pushup": "index.js"
   },
+  "homepage": "https://github.com/code-pushup/cli#readme",
+  "bugs": {
+    "url": "https://github.com/code-pushup/cli/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/code-pushup/cli.git",
+    "directory": "packages/cli"
+  },
+  "contributors": [
+    {
+      "name": "Matěj Chalk",
+      "email": "matej.chalk@flowup.cz",
+      "url": "https://flowup.cz/en"
```

Comment:
Do we need that in the package json?? A contributors section in readme is imho easier to maintain and looks cooler

Reply:
@matejchalk 

> A GitHub profile link would be a better URL for individual contributors than a company website 🤔

Sure, whichever you prefer 🙂

Reply:
@BioPhoton 

> Do we need that in the package json?? A contributors section in readme is imho easier to maintain and looks cooler

![image](https://github.com/code-pushup/cli/assets/6364586/1f81a37d-4fa1-4444-ab95-760620e4fb17)

Currently, we don't need to add `contributors` for the same of the npm page since it has registered you all having write access to the packages (that's my best guess), so we can remove it.

![image](https://github.com/code-pushup/cli/assets/6364586/054c5dcf-66e5-4967-b73f-cb6e9d6b4dfa)

For license, website, repo, and bug tracker, I would recommend it so that these links are added to the metadata sidebar on npm pages.

To make it more maintainable by preventing duplication, we could add these properties to the root-level `package.json` file and copy them into the packages in a post-build task.

Reply:
Contributors metadata have been added to the root `package.json` file and is copied via our `PackageJSON` ESBuild pluign.

Reply:
Contributors metadata have been added to the root `package.json` file and is copied via our `PackageJSON` ESBuild plugin.

Reply:
@BioPhoton 

> A contributors section in readme is imho easier to maintain and looks cooler

We could use [All Contributors](https://allcontributors.org/) for this but for npm and other tools, we need `package.json` metadata.